client/src/configs/token.js

```diff
@@ -0,0 +1,3 @@
+const token =
+  'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9wZmEuZm9yZWNhLmNvbVwvYXV0aG9yaXplXC90b2tlbiIsImlhdCI6MTY1MDYzNjIwMiwiZXhwIjo5OTk5OTk5OTk5LCJuYmYiOjE2NTA2MzYyMDIsImp0aSI6ImU3ODNiOTA5NDdhZjZlYjciLCJzdWIiOiJsdWtvbmVsaXMxIiwiZm10IjoiWERjT2hqQzQwK0FMamxZVHRqYk9pQT09In0.aZP7m278J_ToDZFqLUVBARHs161WQf-bjhbwa1y_fIo';
```

Comment:
please don't store hardcoded token in repository

Reply:
Silent auth implemented, removed token file